examples/grommet/src/components/VernaOverride/DropZone.tsx

```diff
@@ -0,0 +1,22 @@
+import type { DropZoneOverloadProps } from '@openfun/verna';
+import { Box } from 'grommet';
+import React from 'react';
+import './styles/DropZone.scss';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+const messages = defineMessages({
+  descriptionDraggingOver: {
+    defaultMessage: 'Drop here',
+    description: 'The description written in the drop zone when dragging over',
+    id: 'components.VernaOverride.DropZone.descriptionDraggingOver',
+  },
+});
+
+export default function DropZone({ isDraggingOver }: DropZoneOverloadProps) {
+  return (
+    <Box align="center" height={isDraggingOver ? '30px' : '15px'} justify="center">
+      {!isDraggingOver && <div className="drop-zone-divider" />}
+      {isDraggingOver && <FormattedMessage {...messages.descriptionDraggingOver} />}
```

Comment:
```suggestion
      {!isDraggingOver ? (
          <div className="drop-zone-divider" />
        ) : (
          <FormattedMessage {...messages.descriptionDraggingOver} />
        )}
```
or even would set the element outside to be clearer, in this idea or with a if /else
const dragging_zone = isDraggingOver ? ...
...
<Box ...>
{dragging_zone}